### PR TITLE
co-597 fix: removing hardcoded diksha labels

### PR DESCRIPTION
**Pull request by vaivk369**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions: Angular 8, Nodejs 12.16.1
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/resourcebundles/data/consumption/en.properties [hunk 1]
```diff
@@ -1392,8 +1392,8 @@ frmelmnts.lbl.contributionRejected=Contributions Rejected
 frmelmnts.lbl.contributionPending=Contributions Pending
 frmelmnts.lbl.addToLibrary= Add to library
 frmelmnts.lbl.viewContent=View Content
-frmelmnts.btn.viewContentOnDiksha=View this content on DIKSHA
-frmelmnts.btn.viewContentOnDikshaError=ERROR: Content is not available on DIKSHA
+frmelmnts.btn.viewContentOnConsumption=View this content on DIKSHA
```
Comment (vinukumar-vs) [inline src/app/resourcebundles/data/consumption/en.properties:1395]: The value is not changed. Only key is changed.
"View this content on DIKSHA"

File: src/app/resourcebundles/data/consumption/en.properties [hunk 2]
```diff
@@ -1392,8 +1392,8 @@ frmelmnts.lbl.contributionRejected=Contributions Rejected
...
+frmelmnts.btn.viewContentOnConsumptionError=ERROR: Content is not available on DIKSHA
```
Comment (vinukumar-vs) [inline src/app/resourcebundles/data/consumption/en.properties:1396]: Value also have "DIKSHA" keyword.

File: src/app/resourcebundles/data/consumption/en.properties [hunk 3]
```diff
@@ -1733,8 +1733,8 @@ messages.imsg.featureNotEnabled = This feature is not enabled yet.
 #My Content
 frmelmnts.lbl.myContent=My Content
 frmelmnts.lbl.totalContributed=Total Contributed
-frmelmnts.lbl.publishedonDiksha=Published on DIKSHA
-frmelmnts.lbl.notPublishedOnDiksha=Not Published on DIKSHA
+frmelmnts.lbl.publishedforconsumption=Published on DIKSHA
```
Comment (vinukumar-vs) [inline src/app/resourcebundles/data/consumption/en.properties:1736]: Value also have DIKSHA keyword

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/shared/services/config/app.config.json [hunk 4]
```diff
@@ -1,7 +1,4 @@
 {
-  "UrlLinks": {
-    "downloadDikshaApp": "https://play.google.com/store/apps/details?id=in.gov.diksha.app"
```
[the remark this answers is not in the document]
Comment (vaivk369) [reply src/app/client/src/app/modules/shared/services/config/app.config.json:3]: was not used anywhere and that is why removed

File: src/app/routes/googleSignInRoutes.js [hunk 5]
```diff
@@ -117,13 +117,13 @@ const getQueryParams = (queryObj) => {
 }
 const getErrorMessage = (error) => {
   if (error === 'USER_NAME_NOT_PRESENT' || _.get(error, 'message') === 'USER_NAME_NOT_PRESENT') {
-    return 'Your account could not be created on DIKSHA due to your Google Security settings';
+    return 'Your account could not be created on Sunbird due to your Google Security settings';
```
Comment (vinukumar-vs) [inline src/app/routes/googleSignInRoutes.js:120]: This should be instance name. It should not hardcode as Sunbird. 
If so for all adopters has to do code change to show their instance name.

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

**Pull request closed without merging**